manual/techdoc.tex

```diff
@@ -258,12 +258,16 @@ \subsubsection{Initialization}
 
 On some platforms a bit of setup work is required. This is done in the \texttt{InitCallstack()} function.
 
+On windows, tracy will attempt to preload symbols at \texttt{InitCallstack()} time. It does this for device drivers and process modules. As this process can be slow when a lot of pdbs are involved, you can set the \texttt{TRACY\_NO\_DBHELP\_INIT\_LOAD} environment variable to "1" to disable this behavior and rely on-demand symbol loading.
```

Comment:
This document is kind of obsolete. You want to include this in `tracy.tex`. Also, captitalize Windows.